src/Test/Reflection/TestAttributeProvider.php

```diff
@@ -0,0 +1,49 @@
+<?php
+declare(strict_types=1);
+
+namespace Neusta\Pimcore\TestingFramework\Test\Reflection;
+
+use Neusta\Pimcore\TestingFramework\Test\Attribute\KernelConfiguration;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @internal
```

Comment:
Why internal? It is a test class and will not be available in vendor right?

Reply:
No, it's in the `Test` namespace of the bundle and will/must be downloaded. This is a helper class that's needed to fetch the attributes from the class/methods.

Reply:
okay